content/news/2024-11-26-FLUX/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Text-to-Image Magic: FLUX Models Now in Galaxy!"
+date: "2024-11-26"
+tease: "Turn your text into stunning visuals with FLUX models, now integrated into Galaxy!"
+hide_tease: false
+authors: Arash Kadkhodaei
+subsites: [global, eu, us, freiburg]
+tags: [tool]
+---
+
+## Using Black Forest Labs' FLUX Models Inside Galaxy: A Guide to Text-to-Image Integration
+
+[The Black Forest Labs (BFL)](https://blackforestlabs.ai/) FLUX series has made significant strides in the world of generative AI, providing robust text-to-image capabilities
```

Comment:
You could start this here like ... in previous news we have written about speech-to-text tool whisper and the ChatGPT tool in Galaxy [link] ... today we would like to talk about a new tool to create images from texts. ...

Reply:
Wait, sorry, @arash77 there are better way to link to internal stuff. relative links are possible.